### PR TITLE
Improve add a pallet

### PR DESCRIPTION
**Pull request by riusricardo**
Enhance chain spec.

### CODE REVIEW

File: tuts/add-a-pallet/v2.0.0-alpha.6/index.md [hunk 1]
```diff
@@ -444,7 +444,7 @@ Genesis configurations are controlled in `node/src/chain_spec.rs`. We need to mo
 **`node/src/chain_spec.rs`**
 
 ```rust
-use node_template_runtime::{ContractsConfig, MILLICENTS};
+use node_template_runtime::{ContractsConfig, ContractsSchedule, MILLICENTS,};
```
Comment (riusricardo) [inline tuts/add-a-pallet/v2.0.0-alpha.6/index.md]:
```suggestion
use node_template_runtime::{ContractsConfig, ContractsSchedule, MILLICENTS};
```

**Review by danforbes — APPROVED** [review]
Thanks, Ricardo!

**Review by JoshOrndorff — DISMISSED** [review]
This reflects the changes from https://github.com/paritytech/substrate/pull/5712 right? It shouldn't be merged into the alpha.6 tutorial because that PR was not included in the alpha.6 release.

Some options to move forward:
* Duplicate the alpha.6 to alpha.7 now and include this change there. But don't include the alpha.7 tutorial in the versions list until the alpha.7 node template is cut.
* Just leave this PR on ice until alpha.7 is cut.

**Comment by riusricardo** [comment]
Not really @JoshOrndorff this is applicable for alpha.6. It is a better way of configuring the spec file and remove the hacky way.
It still contains the gas calculation.
Alpha 7 will require a bigger rewrite

You can validate the same change for alpha 6 in https://github.com/paritytech/paracon-node

**Comment by JoshOrndorff** [comment]
Oh I see. It eliminates the need to have that mutable config, and makes the contracts config look like all the rest. Good change :+1:

**Pull request merged**